tests/test_other.py

```diff
@@ -958,7 +987,8 @@ def measure_funcs(filename):
             seen = max(measure_funcs('test.js').values())
             high = expected_ranges[outlining_limit][1]
             print(Building.COMPILER_TEST_OPTS, outlining_limit, '   ', low, '<=', seen, '<=', high)
-            assert low <= seen <= high
```

Comment:
I kind of liked having found a place I can use python's `x < y < z` notation ;)

But agreed it's better to change this to test-friendly output.

Reply:
Yes. What is more python's unittest distinguishes between `self.assertXXXX()` failures and other types of assert's like this.  One is called `failure` and one is called and `error`.  In generally we always want to signal tests failures with `self.assertXXX`  ..  i think.. although it probably doesn't matter a whole lot.